network/vpack/parse.go

```diff
@@ -0,0 +1,292 @@
+// Code generated by gen.go; DO NOT EDIT.
```

Comment:
shouldn't we add `go:generate` into gen.go in order to force build process the gen source to its product verification?